test/basics/demo-complex.html

```diff
@@ -0,0 +1,18 @@
+<script type='text/stache' can-autorender id='demo-html'>
+	{{#each items}}
+		<li>{{name}}</li>
+	{{/each}}
+</script>
+</script>
+
+<script src="../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
+import "can-view-autorender";
+import "can-stache-bindings";
```

Comment:
[can-stache-bindings](https://canjs.com/doc/can-stache-bindings.html) is for stuff like `{prop}="value"`

What specifically did you want to test? To me, it looks like “complex” means loading modules with StealJS, so I would probably simplify the file to something like this:

```html
<script src="../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
import stache from "can-stache";
var renderer = stache("<h1>Hello {{subject}}</h1>");
var fragment = renderer({subject: "World"})
document.body.appendChild(fragment)
</script>
```